DataBiosphere/Saturn-UI
# Template Workspace when importing data [SATURN-1149]

Add ability for user to import data directly to template workspace.

To Test:
Follow the flow of importing data and once you get to the screen with 'Destination Workspace' add query params to the url and refresh. (To see the json file the template workspace list is coming from the firecloud-alerts bucket in dev. There are currently two lists in the file with the keys 'Terra' and 'bdC'.)

i.e. add: `&template=Terra&template=bdC` to the url

EDIT: Any workspace that shows in the list now has been validated and should be able to be imported! 
*̶N̶O̶T̶E̶*̶ ̶O̶n̶l̶y̶ ̶t̶h̶e̶ ̶`̶j̶i̶m̶m̶y̶ ̶r̶e̶a̶d̶s̶`̶ ̶t̶e̶m̶p̶l̶a̶t̶e̶ ̶w̶o̶r̶k̶s̶p̶a̶c̶e̶ ̶w̶i̶l̶l̶ ̶w̶o̶r̶k̶ ̶c̶u̶r̶r̶e̶n̶t̶l̶y̶ ̶a̶s̶ ̶t̶h̶e̶ ̶o̶t̶h̶e̶r̶ ̶w̶o̶r̶k̶s̶p̶a̶c̶e̶s̶ ̶d̶i̶d̶ ̶n̶o̶t̶ ̶h̶a̶v̶e̶ ̶t̶h̶e̶ ̶r̶i̶g̶h̶t̶ ̶p̶e̶r̶m̶i̶s̶s̶i̶o̶n̶s̶.̶

Tested:
- With query params of 'Terra', 'bdC' and 'fakeNotReal' in different combinations
- Tried all the flows successfully: making a new workspace, starting with existing and starting with a template
- Successfully started with a template on Terra and DataStage
